### PR TITLE
New function "Select all" in popup menu of textarea

### PR DESCRIPTION
**Pull request by marcoM32**
Hi jreepad Team,
I've added a new feature, now in the popup menu of textarea is present the "select all" function.
I have also reviewed the resource files.

Thanks and best regards
Marco

### CODE REVIEW

File: src/jreepad/editor/EditPopupHandler.java [hunk 1]
```diff
@@ -17,21 +17,23 @@
...
- * @version 1.0
+ * @version 1.1
  */
 public class EditPopupHandler
         extends MouseAdapter {
     private JPopupMenu popupMenu = new JPopupMenu();
     public static final String ITEM_CUT = "cut";
     public static final String ITEM_COPY = "copy";
     public static final String ITEM_PASTE = "paste";
+	public static final String ITEM_SELECTALL = "selectall";
```
Comment (danstowell) [inline src/jreepad/editor/EditPopupHandler.java]: Here and elsewhere I see whitespace errors (tabs vs spaces). Could you please ensure your whitespace usage matches the rest of the file?

File: src/jreepad/lang/JreepadStrings.properties [hunk 2]
```diff
@@ -150,7 +152,7 @@ PREFS_EXPORTTYPE_TEXTILEHTML=Textile format
 
 AUTOSAVE=Autosave
 AUTOSAVE_EVERY=Autosave every
-AUTOSAVE_MINUTES= minutes
+AUTOSAVE_MINUTES=\ minutes
```
Comment (danstowell) [inline src/jreepad/lang/JreepadStrings.properties]: aha thanks

**Review by danstowell — CHANGES_REQUESTED** [review]
Thanks for this contribution. Looks good to me, just the whitespace needs fixing before merge

**Review by marcoM32 — COMMENTED** [review]
I apologize for that mistake, I corrected the problem.

regards
Marco

**Review by danstowell — APPROVED** [review]

**Comment by danstowell** [comment]
Thank you

**Pull request merged**